### DISCUSSION TITLE
Allow hiding some buttons from the main window

### DISCUSSION THREAD

**Discussion by AbraxasDMG** [audacious-media-player/audacious#116] [Ideas]
Hi everyone.

Would it be possible to allow to hide the buttons for Search Library, Open and Add Files from the top left hand area of the main window?

Thank you.
Cheers!

**Comment by radioactiveman** [comment]
This is not supported yet.

Why do you want to hide the buttons? I assume you don't use them often and you want a cleaner/simpler interface?

**Comment by AbraxasDMG** [comment]
Hi.
Thanks.

Well, kind of.
Without these specific buttons, aside from the menu button in the main
window, the interface would provide only controls related to
controlling/playing songs.

Searching the library, and adding files through a file dialogue might not
be something that is a main way of interaction with the player for
substantial number of users (from direct and indirect experience, I'd say
that the approach of dragging desired songs over the playlist before
playing them is used quite a lot).

For that reason I though that the ability to show/hide these buttons could
be justified.
